src/Capgemini.PowerApps.PackageDeployerTemplate/Services/FlowActivationService.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Capgemini.PowerApps.PackageDeployerTemplate.Adapters;
+using Microsoft.Extensions.Logging;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Query;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Capgemini.PowerApps.PackageDeployerTemplate.Services
+{
+    public class FlowActivationService
+    {
+        private readonly ILogger logger;
+        private readonly ICrmServiceAdapter crmSvc;
+
+        public FlowActivationService(ILogger logger, ICrmServiceAdapter crmSvc)
+        {
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            this.crmSvc = crmSvc ?? throw new ArgumentNullException(nameof(crmSvc));
+        }
+
+        public void ActivateFlows(IEnumerable<string> flowsToIgnore, IEnumerable<string> solutions)
```

Comment:
`flowsToDeactivate` might make the side-effects of this method clearer

Reply:
Agreed